src/components/App.js

```diff
@@ -2,10 +2,9 @@ import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import { Home } from "./Home";
 import { RandomCat } from "./RandomCat";
 import { About } from "./About";
-import { ChakraProvider } from "@chakra-ui/react"
+import { ChakraProvider } from "@chakra-ui/react";
 import "../App.css";
 
-
 function App() {
   return (
     <ChakraProvider>
```

Comment:
para no tener que repetir codigo y agregar el Header en cada pantalla lo podemos poner aca
```suggestion
    <ChakraProvider>
      <Header />
```